app/src/main.cpp

```diff
@@ -83,6 +101,44 @@ void loop() {
 
   monitorAudioEngine();
 
+  // Run display controls
+  disp.displayVol();
+  disp.displayPan();
```

Comment:
Similar to the above, the "pan" value is already updated to a value between 0 and 1 in the `adjustPan` function above. So this function shouldn't need to read the knob again, and it can just read the global `pan` variable. This way, if we change the logic for reading the knob, it will only need to be changed in one place.